apps/files/src/components/FilesLists/StatusIndicators/StatusIndicators.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+  <div class="uk-flex uk-flex-middle">
+    <DefaultIndicators
+      v-if="displayDefaultIndicators"
+      :item="item"
+      :currentFolderPath="currentFolderPath"
+      :class="{ 'uk-margin-xsmall-right' : customIndicators }"
+      @click="openSidebar"
+    />
+    <template v-if="customIndicators">
+      <component
+        v-for="(indicator, index) in customIndicators"
+        :is="indicator"
+        :key="index"
+        :item="item"
+        :currentFolderPath="currentFolderPath"
+      />
+    </template>
+  </div>
+</template>
+
+<script>
+import { mapGetters } from 'vuex'
+
+const DefaultIndicators = () => import('./DefaultIndicators.vue')
+
+export default {
+  name: 'StatusIndicators',
+
+  components: {
+    DefaultIndicators
+  },
+
+  props: {
+    item: {
+      type: Object,
+      required: true
+    },
+    currentFolderPath: {
+      type: String,
+      required: true
+    }
+  },
+
+  computed: {
+    ...mapGetters(['configuration', 'customFilesListIndicators']),
+
+    displayDefaultIndicators () {
+      return !this.configuration.theme.filesList.hideDefaultStatusIndicators
```

Comment:
beware in case "theme" and "theme.filesList" are not defined

maybe we need some utility method that we can query with a string ? or wasn't there an ES6 operator to accomodate for such situations ?

Reply:
This is coming from the store with a default value so it would have to be either manually deleted or it would be a bug. I haven't put there a check for it exactly because of the bug case so we would then see an error if something is wrong.